src/battle-animations.ts

```diff
@@ -794,6 +794,10 @@ class BattleScene {
 			let lombreCount = 0;
 			for (let i = 0; i < side.pokemon.length; i++) {
 				let pokemon = side.pokemon[i];
+				if (pokemon.speciesForme === 'Xerneas-*') {
+					side.pokemon[i].name = 'Xerneas-Neutral';
+					side.pokemon[i].speciesForme = 'Xerneas-Neutral';
```

Comment:
Hrm, if this works with no negative consequences, I'd be pretty happy about that. Otherwise it might be better to implement this inside `getSpriteData`.

Reply:
Hm. Yeah, this seems safe enough. We just need that edit above and this should be good.